app/controllers/articles_controller.rb

```diff
@@ -0,0 +1,5 @@
+class ArticlesController < ApplicationController
+  def index
+    @pages = Page.published.order('created_at DESC')
+  end
+end
```

Comment:
Instead of an `ArticlesController`, I'd feel better if we add this as a method on the `PagesController` (e.g. `PagesController#feed`).

Reply:
ok